results/cmd/api/main.go

```diff
@@ -65,6 +65,7 @@ type server struct {
 	env *cel.Env
 	db  *sql.DB
 }
+type newID func() string
```

Comment:
Doesn't look like this is being used. Remove.

Reply:
done